lua/gruvbox_crisp/groups.lua

```
@@ -0,0 +1,283 @@
+    ["@keyword.directive"]   = { fg = p.preproc },
+    ["@operator"]       = { link = "Operator" },
+    ["@punctuation"]    = { fg = p.punct },
+    ["@punctuation.bracket"] = { fg = p.punct },
+    ["@punctuation.delimiter"] = { fg = p.punct },
+    ["@punctuation.special"]   = { fg = p.operator },
+    ["@type"]           = { link = "Type" },
+    ["@type.builtin"]   = { link = "Type" },
+    ["@namespace"]      = { fg = p.number }, -- bright purple
+    ["@type.parameter"]  = { link = "Type" },
+    ["@attribute"]      = { fg = p.decorator },
+    
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)